Chat/ClientApp/src/core/sideEffects.ts

```diff
@@ -633,6 +646,19 @@ const getToken = async () => {
   }
 };
 
+const refreshTokenAsync = async (userMri: string) => {
```

Comment:
Let's please not use the term `Mri` in any of our open-source code. We've worked hard to hide MRIs to the world, otherwise we would need to explain and document them.

Rename to `userIdentity`?